### ISSUE TITLE
Posix/OpenLDAP (RFC2307) does not work

### ISSUE THREAD

**Issue by ozawa-hi** [nilsteampassnet/TeamPass#2709]
### Steps to reproduce
1. Setup Posix/OpenLDAP(RFC2307) as instructed in the following teampass page with "@" in the "LDAP account suffix for your domain".
https://teampass.readthedocs.io/en/latest/install/ldap/
2 Enter Test configuration. 
3. Press "Start" icon.

### Expected behaviour
Tell us what should happen
Test should be successful.

### Actual behaviour
Tell us what happens instead
Following error message is displayed:
"After authenticate: Invalid DN syntax"

However, if I remove the "@" from the LDAP account suffix, I get "Ater authenticate: Invalid credentials". So, there probably shouldn't be a "@" as is written in the teampass documentation.

Nevertheless, teampass still does not work with "Posix/OpenLDAP (RFC2307)" even though I can get it to pass the test using "Posix/OpenLDAP(RFC 2307) Search Based".

### Server configuration
**Operating system**:
Ubuntu 18.04

**Web server:**
Apache2
**Database:**
MySQL
**PHP version:**
7.2
**Teampass version:** 
2.1.27.36
**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**
PLEASE attach to this issue the file `/includes/config/tp.config.php`.

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
```
Insert your webserver log here
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by ozawa-hi** [comment]
I got it to work. There's a bug with the "Test current configuration" in the LDAP setting page.
Following is the settings I used:
LDAP server type: Posix/OpenLDAP (RFC2307)
LDAP account suffix for your domain:   (leave empty!)
LDAP base DN for your domain: dc=ozawa,dc=com
LDAP domain controller(s):127.0.0.1
LDAP port:389

When I execute "Test current configuration" with the above settings, it will fail with "Invalid DN syntax" but if I logout and login with an user in LDAP, a new user will be created and the user will be able to login.
